### ISSUE TITLE
issues about training

### ISSUE THREAD

**Issue by sylvia1127** [cddlyf/GCANet#14]
Hi! I used the training code you provided with RESIDE ITS training dataset and SOTS indoor testing dataset. I have trained 140 epochs, and the average loss is close to 12, PSNR is close to 28.  Do you have more experimental details ?
